### ISSUE TITLE
Fix language codes

### ISSUE THREAD

**Issue by itsnotsaved** [directus/directus#10633]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

I noticed that currently directus store available languages with country code. e.g. af-ZA Please store them with language code by default and use country code only for especially required or significant contribution receive languages.

https://github.com/directus/directus/blob/main/app/src/lang/available-languages.yaml

When projects follow this practice, then people living in other countries or lands (or as minor groups in different regions) that use same language try to create separate versions for them only for political purposes. (or propose versions only has some synonyms difference or second versions contain many borrowed words from other languages)
Basically this harm to original languages with time and otherhand affect to the localization speed around world due split into groups

### To Reproduce

-

### Errors Shown

-

### What version of Directus are you using?

0

### What version of Node.js are you using?

0

### What database are you using?

0

### What browser are you using?

0

### What operating system are you using?

0

### How are you deploying Directus?

0
